spec/pagseguro/transaction_request_spec.rb

```diff
@@ -120,8 +120,7 @@
       let(:raw_xml) { File.read("./spec/fixtures/invalid_code.xml") }
 
       it "does not create a transaction request" do
-        expect(transaction_request.create).to be_falsy
-        expect(transaction_request.code).to be_nil
+        expect { transaction_request.create }.not_to change { transaction_request.code }
```

Comment:
I don't understand what this spec is testing, because you're just checking if the code attribute was changed. How can I confirm whether my transaction request was created or not?